segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DebugRecoveryProcessor.java

```diff
@@ -131,6 +141,66 @@ protected void recoverOperation(DataFrameRecord<Operation> dataFrameRecord, Oper
         }
     }
 
+    /**
+     * The overridden behaviour in this method is to choose between  using a DebugDataFrameReader and the regular DataFrameReader.
```

Comment:
Extra space in `between using`.

Reply:
this will be removed based on above comment